data/json/mapgen/basement/basement_game.json

```diff
@@ -60,6 +60,9 @@
         "s": "f_sofa",
         "t": "f_table"
       },
+      "toilets": {
+        "&": {}
```

Comment:
It also appears there is no "i" symbol, so the sink was had the solid rock too... I'll fix this.